proto/viam/app/data/v1/data.proto

```diff
@@ -80,6 +80,14 @@ message Filter {
   repeated string org_ids = 11;
   repeated string mime_type = 12;
   CaptureInterval interval = 13;
+  TagsOption tags_option = 14;
+}
+
+enum TagsOption {
```

Comment:
I am very open to renaming this to something other than "TagsOption," since I don't think this is the most descriptive